packages/core/src/index.js

```diff
@@ -17,6 +17,7 @@ import setSendBox from './actions/setSendBox';
 import setSendTimeout from './actions/setSendTimeout';
 import setSendTyping from './actions/setSendTyping';
 import setSendTypingIndicator from './actions/setSendTypingIndicator';
+import setSuggestedActions from './actions/setSuggestedActions';
```

Comment:
Could we create another action named `clearSuggestedActions` here and expose it instead?

Doesn't like the idea of allowing developers to put whatever suggested actions button they want in the UI without communicating with the bot. In this list, except `markActivity`, other actions we expose here doesn't have that level of impact.